scripts/jenkins/cloud/ansible/crowbar-update.yml

```diff
@@ -47,25 +47,26 @@
               delegate_to: "{{ target_host }}"
           vars:
             inventory_hostname: "{{ target_host }}"
-          loop: "{{ [cloud_env]+groups['cloud_virt_hosts'] }}"
+          loop: "{{ groups['cloud_virt_hosts'] }}"
```

Comment:
Just a question. Is `cloud_env`(admin node) in `cloud_virt_hosts`?

Reply:
no, the admin node is explicitly excluded from the `cloud_virt_hosts` group [here](https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/cloud/ansible/_create-vcloud-inventory.yml#L9)